mantle/kola/tests/misc/network.go

```diff
@@ -172,3 +185,247 @@ func NetworkInitramfsSecondBoot(c cluster.TestCluster) {
 		c.Fatal("networkd started in initramfs")
 	}
 }
+
+var (
+	defaultLinkConfig = `[Link]
+          NamePolicy=mac
```

Comment:
Note here in these multiline strings we're encoding all this leading whitespace. But heh, I guess none of these files care about that.